spec/meta-schema/meta-schema.spec.ts

```diff
@@ -939,6 +1024,84 @@ describe('Meta schema API', () => {
         });
     });
 
+    it('can query raw localization of types', async () => {
```

Comment:
maybe add a test for a field where only one of the languages is present, to make sure the value omits this key then?

Reply:
added a test for a field on the root entity type and a value object, both with only german translations